src/DynamoCore/Extensions/LinterExtensionBase.cs

```diff
@@ -0,0 +1,259 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Dynamo.Graph.Nodes;
+using Dynamo.Graph.Workspaces;
+using Dynamo.Linting;
+using Dynamo.Linting.Rules;
+
+namespace Dynamo.Extensions
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public abstract class LinterExtensionBase : IExtension
```

Comment:
I think we can remove the Base word and just leave it LinterExtension

Reply:
think its nice to easily see that this is only serving as a base for other classes? I believe that this is also the pattern used for other base classes in Dynamo like: `ModelBase`, `ViewModelBase`, `ViewExtensionBase`

But happy to change it if we think that is better?

Reply:
No, you are right, let's keep it as it is :)